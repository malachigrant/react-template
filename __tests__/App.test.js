/** @jsx jsx */
import { create } from 'react-test-renderer';
import { jsx } from '@emotion/core';

import App from '../src/App';

describe('App', () => {
  it('renders correctly', () => {
    const tree = create(<App />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
