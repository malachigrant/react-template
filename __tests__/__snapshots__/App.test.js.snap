// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
Array [
  <style
    dangerouslySetInnerHTML={
      Object {
        "__html": ".css-15tlx1l-App{margin:1em auto 0 auto;padding:1em 2em;border-radius:0.2em;}.css-15tlx1l-App:hover{background-color:#000000;color:#ffffff;}",
      }
    }
    data-emotion-css="15tlx1l-App"
  />,
  <div
    className="css-15tlx1l-App"
  >
    Welcome to my template!
  </div>,
]
`;
